### PR TITLE
docs(resources): Add New Section in Docs for External Resources

### PR DESCRIPTION
**Pull request by pertrai1**
**Description:**

This adds a new section on the left hand navigation for External Resources. It is a starting collection of resources that people can use to find more content available on the web. Hopes are that it grows over time to be the one place people can come and find everything they need for information regarding RxJS.

### CODE REVIEW

**Comment by mattyao1984** [comment]
I guess this is a new feature to the doc website and it looks awesome. My suggestion is to have several categories for the resources such as videos, tutorials, blogs etc.

**Comment by pertrai1** [comment]
@mattyao1984 that sounds like a great idea. going off of the categories, which ones do you see being needed for this to be a good resource to start with? I want to get this off the ground and then have smaller PR's towards it. If that does not sound good, I can work on it for a bit more until I find enough resources

**Comment by JWO719** [comment]
Hi @pertrai1 basically this is an awesome PR, thank you very much. Personally I would prefer something similar to the way we render the teams page. There we have a json file for the data and an angular component loading the json file and rendering the content. I think this is far more complex, but it also comes with some benefits. It would be easier to add resources and functions like searching or filtering. Also it would be testable and maybe easier to understand for other contributors. I know that this would be need more effort and therefore I would really like to discuss on that. What do you think of that?

File: docs_app/content/external-resources.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# External Resources
+
+This is a collection of resources that can be found to help with the learning process of RxJS. The list is ever growing which will make it a place for you to come to for finding helpful information.
```
Comment (JWO719) [inline docs_app/content/external-resources.md]: Maybe add a hint here, that RxJS isn't responsible for the content of the sites or something like that

**Comment by pertrai1** [comment]
@JWO719 I like your idea and I will take a look at how Teams is constructed. How about closing this PR and we can take up discussion on how best to put together a library of resources for users?

**Comment by JWO719** [comment]
maybe let's discuss this here -> #3895

**Pull request closed without merging**